test/mock_blockindex/mock_blockindex.go

```diff
@@ -0,0 +1,153 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Oh, I forgot `mock_blockindex` is missing in the ./test folder. Actually, this can be generated by adding cmds in `./misc/scripts/mockgen.sh` and `make mockgen` under root dir